### ISSUE TITLE
comment-format check-uppercase false positive

### ISSUE THREAD

**Issue by Fraegle** [palantir/tslint#649]
``` JavaScript
// If I have a comment that is too long for
// one line, it's OK for the second line to start lower case. 
```

**Comment by JKillian** [comment]
Agreed with you in theory, I'd recommend just disabling the rule if this happens often. Problem is we have no way to distinguish it from

```
// A nice one-line comment
// Another nice one-line comment
```

**Comment by blakeembrey** [comment]
Just run into this too. For `check-uppercase`, it doesn't matter if the second direct line comment is uppercase or lowercase. I think this should be a default, but for backward compatibility I'd love to see `allow-trailing-lowercase` or something.

**Comment by sinedied** [comment]
+1

**Comment by MichaelTontchev** [comment]
Any reason why this hasn't been addressed yet?
